projects/plugins/jetpack/modules/google-analytics/classes/wp-google-analytics-universal.php

```diff
@@ -188,6 +195,76 @@ public function maybe_track_purchases( $command_array ) {
 		return $command_array;
 	}
 
+	/**
+	 * Process purchase tracking options for the universal Google Analytics queue (ga) commands.
+	 *
+	 * May also update post meta to indicate the order has been tracked.
+	 *
+	 * This method is different from maybe_track_purchases in HPOS support.
+	 *
+	 * @see https://github.com/woocommerce/woocommerce/wiki/High-Performance-Order-Storage-Upgrade-Recipe-Book
+	 *
+	 * @since $$next-version$$
+	 * @param array $command_array Array of commands.
+	 * @return array `$command_array` with additional commands conditionally added.
+	 */
+	public function maybe_track_hpos_purchases( $command_array ) {
+		global $wp;
+
+		$order_id = isset( $wp->query_vars['order-received'] ) ? $wp->query_vars['order-received'] : 0;
```

Comment:
I'm not sure if you use them in JP but this could be a null coalesce.

Reply:
Yeah, unfortunately we're still at PHP 5.6 minimum, but someday :)